Directory.Packages.props

```diff
@@ -40,7 +40,7 @@
 
     <!-- Test infra -->
     <PackageVersion Include="Appveyor.TestLogger" Version="2.0.0" />
-    <PackageVersion Include="FluentAssertions" Version="8.2.0" />
+    <PackageVersion Include="FluentAssertions" Version="7.2.0" />
```

Comment:
Downgrading to the version under full OSS licence.

https://fluentassertions.com/:
> Versions 8 and beyond are free for open-source projects and non-commercial use, but commercial use requires a [paid license](https://xceed.com/products/unit-testing/fluent-assertions/). Check out the [license page](https://github.com/fluentassertions/fluentassertions/blob/main/LICENSE) for more information. Version 7 will remain fully open-source indefinitely and receive bugfixes and other important corrections.